### PR TITLE
hotfix/PTX-3795 fix an issue when checking px is up

### PR DESCRIPTION
**Pull request by thiguetta**
Signed-off-by: Thiago Gonzaga <tgonzaga@purestorage.com>

**What this PR does / why we need it**:

this PR adds a third check when px reporting up before it really is.

### CODE REVIEW

**Comment by thiguetta** [comment]
tested here https://jenkins.portworx.dev/job/Torpedo/view/Torpedo%20NextPX/job/tp-ocp-candidate-csi/39/

File: drivers/volume/portworx/portworx.go [hunk 1]
```diff
@@ -980,6 +981,35 @@ func (d *portworx) WaitDriverUpOnNode(n node.Node, timeout time.Duration) error
 					Cause: fmt.Sprintf("px is not yet up on node. cause: %v", err),
 				}
 			}
+			opts := node.ConnectionOpts{
+				IgnoreError:     false,
+				TimeBeforeRetry: defaultRetryInterval,
+				Timeout:         defaultTimeout,
+			}
+			out, err := d.nodeDriver.RunCommand(n, "/opt/pwx/bin/pxctl -j status", opts)
```
Comment (mborodin-px) [inline drivers/volume/portworx/portworx.go:989]: in some cases the path could be different, use which command here:
```
[root@torpedo-tp-basic-vol-driver-26-0 cores]# which pxctl 
/usr/local/bin/pxctl
```

**Review by mborodin-px — APPROVED** [review]
LGTM, do not forget to test it

**Comment by thiguetta** [comment]
tested here https://jenkins.portworx.dev/job/Torpedo/view/Torpedo%20NextPX/job/tp-ocp-stable/81/console
now you see
```
19:54:42 [36mINFO[0m[2021-09-30 22:54:42] Using 10.0.156.117:17017 as endpoint for portworx volume driver 

19:54:42 [37mDEBU[0m[2021-09-30 22:54:42] Running command on pod debug-g2sbz [[nsenter --mount=/hostproc/1/ns/mnt /bin/bash -c which pxctl]] 

19:54:42 [37mDEBU[0m[2021-09-30 22:54:42] Running command on pod debug-g2sbz [[nsenter --mount=/hostproc/1/ns/mnt /bin/bash -c /usr/local/bin/pxctl -j status]] 

19:54:42 2021/09/30 22:54:42 Failed to wait for px status on: ip-10-0-156-117.ec2.internal due to err: node ip-10-0-156-117.ec2.internal status is up but px cluster is not ok. Expected: STATUS_OK Actual: STATUS_INIT Next retry in: 10s

19:55:33 [36mINFO[0m[2021-09-30 22:55:32] Using 10.0.156.117:17017 as endpoint for portworx volume driver 

19:55:33 [37mDEBU[0m[2021-09-30 22:55:32] Running command on pod debug-g2sbz [[nsenter --mount=/hostproc/1/ns/mnt /bin/bash -c which pxctl]] 

19:55:33 [37mDEBU[0m[2021-09-30 22:55:33] Running command on pod debug-g2sbz [[nsenter --mount=/hostproc/1/ns/mnt /bin/bash -c /usr/local/bin/pxctl -j status]] 

19:55:33 [36mINFO[0m[2021-09-30 22:55:33] px on node: ip-10-0-156-117.ec2.internal is now up. status: STATUS_OK 

19:55:33 [37mDEBU[0m[2021-09-30 22:55:33] checking if PX pod is up on node: ip-10-0-156-117.ec2.internal 
```

**Review by mborodin-px — APPROVED** [review]
LGTM

**Pull request merged**